proto/publicrpc/v1/publicrpc.proto

```diff
@@ -51,6 +51,7 @@ enum ChainID {
   CHAIN_ID_UNICHAIN = 44;
   CHAIN_ID_WORLDCHAIN = 45;
   CHAIN_ID_INK = 46;
+  CHAIN_ID_MOVEMENT = 49;
```

Comment:
I think this list is missing some of the previously added chains. Would be good to bring it in line.

Reply:
Don't want to cloud this PR with that stuff. Will do a follow-on PR.